### ISSUE TITLE
Design Intent Enhancements

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#23446]
## Issue Description (Design/UX)
Platform Product Support hosted a [Design Intent workshop](https://app.zenhub.com/workspaces/vsp---product-support-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/22153) on 4/12 ([MURAL board](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1617213449512/58a64f86488a29ceeef4f9c0ba9a9ab14f60f1c5)). While the team captured valuable and helpful feedback from both platform and OCTO-DE practice area leads, by the end of the hour long workshop, it became clear that there was some misalignment in our expectations of the Design Intent.

Platform team members currently review the VFS product prior to the Design Intent meeting. They then come to the meeting prepared to give feedback, offer suggestions, and answer any questions. The feedback they provide is based on the existing UX, however feedback is only applicable to the UI. When the OCTO-DE Design Lead (Ryan Thurlwell) attends the Design Intent meeting, he is not limited to UI feedback and will sometimes provide feedback on the UX. On some occasions, the VFS team will make changes to the UX. While platform is appreciative of Ryan's feedback, it can invalidate the feedback that platform had already provided (either orally or via feedback ticket). 

Platform is proposing re-ordering the events of the Design Intent to better serve Platform Product Support teams members and OCTO-DE Design Lead.

## Issue Description (Sitewide Content)
Sitewide Content introduced a new process recently where they are providing content guidance to VFS teams. VFS teams meet with them early on in their build, and Sitewide Content creates a Content Brief to help the VFS team understand and build a product supporting the content flow. 

Sitewide Content is not currently working/supporting all VFS teams.. It is challenging for Platform team to know what content support should be provided. If the VFS team has met with Sitewide Content, VFS teams will be asked to provide the Content Brief as an artifact for review. If the VFS team has not met with Sitewide Content, Platform is unsure if they are going to soon or if they won't be at all.

## Tasks (design/UX)
- [x] Add a question to the Design Intent template to ask if the Product has been reviewed by the OCTO-DE Design Lead (work with Ryan on language)
- [ ] If product has been reviewed by OCTO-DE Design Lead, their attendance will not be required at Design Intent.
- [ ] If product has not been reviewed by OCTO-DE Design Lead, their attendance will be required at Design Intent
- [x] If feedback from OCTO-DE Design Lead at Design Intent meeting doesn't impact the Veteran UX, Platform will provide written github feedback tickets by COB next business day (AKA "go" decision).
- [ ] If feedback from OCTO-DE Design Lead at Design Intent meeting impacts the Veteran UX, team will still provide feedback tickets and a 2nd Design Intent meeting will be recommended (aka "no-go decision")

## Tasks (Sitewide Content)
- [x] Add a question to the Design Intent template to ask if the team has met with Sitewide Content team for content support.
- [x] If yes, ensure VFS team provides Content Brief artifact
- [ ] If no, determine if the team has not yet met with Sitewide Content OR if they will not be working with Sitewide Content for the product.
   - [ ] If the team has not yet met with Sitewide Content for the product, Platform content support will not be provided at Design Intent.
   - [ ] If the team will not be working with Sitewide Content for the product, Platform content support will be provided at Design Intent.
- [x] Modify internal process to include Randi Hecht as an attendee for Design Intent with Danielle Thierry and Beth Potts as optional

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @shiragoodman. Thanks very much for the meeting and for sharing these next steps.

These tasks look good for info-gathering purposes so VSP understands teams' involvement with sitewide content at this stage.

@bethpottsVADEPO and I did want to clarify 2 things:

**1) Platform content involvement in design intent.** Our understanding from the meeting was that we were all aligning around the thinking that design intent is more for a UX/wireframe type of review and that VSP content likely wouldn't be reviewing teams' actual content at this stage. Is that correct?

**2) Content brief.** We want to make sure we're aligned on expectations for the content brief. The brief will be a living, evolving document built for and with the product team and SME input during product development. We're working collaboratively with the designers as they build the product, often working on content across multiple weeks of office hours and tickets.

VSP should definitely have access to the content brief from the start and as it evolves. But at the design intent stage, the content brief would likely be very high level and therefore wouldn't be all that helpful for the VSP content reviewer. It will likely be helpful at the  midpoint review. It will be most helpful at the staging review stage, where VSP content would be checking final content to ensure that it aligns with VA.gov standards (including specific product content standards outlined in the brief).

**Comment by shiragoodman** [comment]
Hi @DanielleThierryUSDSVA

1) Re: your comment "VSP content likely wouldn't be reviewing teams' actual content at this stage. Is that correct?" the response is yes and no. If we know that the team is working with Sitewide Content, we will not provide content feedback at the Design Intent. We would however, still like the VFS team to share their Content Brief (if started) and any other relevant documentation. If the team is not currently working with Sitewide Content, our action really depends on the following questions:
- will the VFS team be working with Sitewide Content eventually? Ie, they have yet to engage with Sitewide Content.
- will the VFS team NOT be working with Sitewide Content on this product?
If the team has yet to engage with you, we will not be providing content feedback.
If the team will not be engaging with you, we would like to be able to provide content feedback.
These decisions were made because we are under the assumption that your process is still a pilot, and you won't be supporting every team. If this is not true, please let me know. Also, we understand that is likely that the VFS team won't know if they'll be working with you or not. This is where we need Sitewide Content to keep us in the loop on what teams/products you're supporting.

2) Re: your comments on the Content Brief  - got it. Thanks for sharing this info. I think it would be helpful for Peggy, as well as the rest of my team, if we could establish a more reliable process for Sitewide Content to share information with our team.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @shiragoodman.
 
Re: 1) We understand what the different paths are, and yes, we're on board with keeping you in the loop. What we were asking is whether there needs to be _any_ content review at the design intent stage. We thought coming out of the meeting that the thinking was that design intent is too early for getting into the content details. That's what we were asking about here.

Re: 2) Not sure what you mean by more reliable? The brief will have the start of information at the beginning of the project. Then it will have more information as the project evolves. So all we're saying is that at the design intent stage, there won't be all that much information in the brief yet. That's just the nature of an evolving product. 

However, we seem to be missing each other in how we're understanding content process and reviews overall, so we definitely need to get to a better shared understanding. 

Here's how we see it:
- Beth and I are providing support to product teams. This looks different depending on the product and the team's needs and timelines. It's a fluid process because it needs be in order to support the teams.
- Peggy is reviewing product content to make sure it aligns with the basics of VA.gov and plain language content standards.
- Any content guidance that Beth and I provide to teams will align with VA.gov and plain language standards.
- So Peggy should still be able to do her reviews, using the existing style guide, for all products. The only real difference she should see is that  the content Beth and I have been involved in will already be aligned with the standards. So she'll have less to flag.
- Beth and I are also working to continue to refine content standards in our role as OCTO content leads. We will continue to document and share those refined standards so Peggy has them to check against.
- The brief is simply an additional document for product teams that will also be helpful to Peggy. It will capture some basics about content standards (that Peggy already knows and has documented) and some product-specific terms and other information as needed. It's never been intended to capture all of our content feedback, and it doesn't need to. Peggy can still do her checkpoints even without a content brief because she's checking against the existing style guide and standards.

Let's set up a call for next week. We'd like to learn more about how our feedback has been causing problems to date so we can better understand and address any issues.

cc @bethpottsVADEPO

**Comment by shiragoodman** [comment]
@DanielleThierryUSDSVA @bethpotts 

1) I don't think I did a good job articulating this, so my apologies. Yes, Design Intent is too early to provide content feedback, especially if the UX is still being worked on. However, not all teams come to Design Intent with a wireframe. For iterations to existing products, teams will likely have a more high fidelity prototype, and sometimes there will be content to provide feedback on. 

2) let me try again. We will be making changes to our templates to ask VFS teams about their engagement with Sitewide Content. Hopefully that change will help give us the info we need. But we currently don't know what teams you're working with and your process has been somewhat of a "black box" from our perspective. Maybe this is because we have yet to engage with any VFS teams who have a brief to share with us? 

I agree that it would be helpful to get together and discuss. I'll take a look at both of your calendars and will find a time that works for us. I'll include Rachael and Peggy too. Thanks!

**Comment by rroueche** [comment]
@DanielleSoCompany I'll find a time for you and @bethpotts and I to sync on the OCTO-DE side about this - I think we have a couple things we can get alignment on that way, and then come back to discussions with the full team.

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @shiragoodman and @rroueche. Sounds good and looking forward to talking more!
